src/command_modules/azure-cli-container/azure/cli/command_modules/container/commands.py

```diff
@@ -16,4 +16,5 @@ def load_command_table(self, _):
                          table_transformer=transform_container_group)
         g.custom_command('delete', 'delete_container', confirmation=True)
         g.custom_command('logs', 'container_logs', client_factory=cf_container_logs)
+        g.custom_command('exec', 'container_exec')
```

Comment:
This should be a verb, not a platform-specific command name. I would suggest something like "run" or "run-command" (which has precedent in the VM module).

Reply:
Exec is the standard in the Kubernetes/Docker community.